README.md

```diff
@@ -29,6 +29,7 @@ This plugin is compatible with CSI version 1.3.0.
 | Filestore CSI Driver\Kubernetes Version | 1.12 | 1.13 | 1.14 | 1.15 | 1.16+ |
```

Comment:
We can simplify the table by making a combined column 1.12 - 1.13 and just have a 1.15+ column

Although what versions are we testing 0.3.0 on? Maybe we should only advertise support for the versions where we're testing. k8s only supports 3 versions back anyway and since it's not going to matter for GKE maybe we should drop 1.14 support so we aren't on the hook for any bugs that come up.

There aren't any CRDs or anything that we depend on for older versions? The PD CSI GA version is only 1.17+ but I'm not sure why.

I guess topology is one thing that's changing (GA in 1.17), but this driver supports both that and the beta?

Reply:
1.17 is due to provisioner topology, snapshot supports. I have updated the table